codelab-master/apps/playground/src/app/quiz/containers/introduction/introduction.component.html

```diff
@@ -0,0 +1,40 @@
+<ng-container *ngFor="let quiz of quizData">
+  <mat-card *ngIf="quiz?.quizId === quizName">
+    <mat-card-header>
+      <div mat-card-avatar class="header-image"></div>
+      <mat-card-title i18n>{{ quiz?.milestone }} Quiz</mat-card-title>
+      <mat-card-subtitle i18n>
+        <span>How well versed are you in {{ quiz?.milestone }}?</span>
+        <span>Take the quiz and find out!</span>
+      </mat-card-subtitle>
+    </mat-card-header>
+
+    <div mat-card-image class="quiz-topic-image"
+         [ngStyle]="{ 'background': 'url(' + imagePath + quiz?.image + ') no-repeat center top',
+                      'background-size': '400px 280px' }">
+    </div>
+
+    <mat-card-content>
+      <span i18n><strong>{{ quiz?.summary }}</strong>  This quiz is composed of
+        <strong>{{ quiz?.questions.length }}</strong> timed multiple-choice question(s). Taking this quiz should
```

Comment:
why is quiz optional at all?
Let's have it required

Reply:
ok I fixed the intro component. will check the other template files when I have time.